packages/block-library/src/post-excerpt/edit.js

```diff
@@ -84,6 +91,23 @@ export default function PostExcerptEditor( {
 			}
 		/>
 	);
+	const excerptContent = isEditable ? (
+		<RichText
+			className={
+				! showMoreOnNewLine &&
+				'wp-block-post-excerpt__excerpt is-inline'
+			}
+			aria-label={ __( 'Post excerpt text' ) }
+			value={
+				excerpt ||
+				postContentExcerpt ||
+				( isSelected ? '' : __( 'No post excerpt found' ) )
+			}
+			onChange={ setExcerpt }
+		/>
+	) : (
+		excerpt || postContentExcerpt || __( 'No post excerpt found' )
```

Comment:
Nit: This value could potentially be extracted in a variable as it's used twice.

Reply:
Actually it's not because in `editable` we also check if `isSelected`. By extracting it I would also add an `isSelected && isEditable` check and I think it will be less readable. What do you think?

Reply:
Right I missed that part.